src/main/java/de/chojo/repbot/listener/ReactionListener.java

```diff
@@ -90,6 +94,33 @@ public void onGuildMessageReactionAdd(@NotNull GuildMessageReactionAddEvent even
         reputationManager.submitReputation(event.getGuild(), event.getUser(), message.getAuthor(), message, null, ThankType.REACTION);
     }
 
+    @Override
+    public void onGuildMessageReactionRemoveEmote(@NotNull GuildMessageReactionRemoveEmoteEvent event) {
+        var optGuildSettings = guildData.getGuildSettings(event.getGuild());
```

Comment:
Yeah, fine. A little duplicated code but nothing that would require refactoring.